### PR TITLE
Csv readme

### PR DESCRIPTION
**Pull request by Deep1998**

### CODE REVIEW

File: sources/csv/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+]
+```
+**CAVEATS:**
+- File patterns do not accept regular expressions. Provide the path inside 
+double quotes.
+
+### CSV File Format
+- Harbourbridge checks the first row and matches it with the spanner columns
+to check if the first row is a permutation of the Spanner table's column names.
+If a match is not found, then Harbourbridge assumes the order of data in csv
```
Comment (bharadwaj-aditya) [inline sources/csv/README.md:70]: do we have sanity checks to see the number of columns match ?

**Review by bharadwaj-aditya — COMMENTED** [review]
Just one comment. I might missed it in the CSV code PR. Please check that bit

File: sources/csv/README.md (hunk 1 above)
Comment (Deep1998) [reply sources/csv/README.md:70]: Good catch, I missed this check. Added

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

**Pull request merged**